engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/function/Function.java

```diff
@@ -378,4 +378,23 @@ public boolean isThunk() {
   public boolean isFullyApplied() {
     return schema.isFullyApplied();
   }
+
+  @ExportMessage
+  String toDisplayString(boolean sideEffects) {
+    return toString();
+  }
+
+  @Override
+  @CompilerDirectives.TruffleBoundary
```

Comment:
This change is unrelated, but hopefully useful. Prints name of a function and its location in source code rather than `...Function@493842143fae43`.